src/mmw/js/src/data_catalog/templates/checkboxFilter.html

```diff
@@ -1,11 +1,11 @@
 <div class="checkbox data-catalog-filter-checkbox">
-    <h5>{{ label }}</h5>
      <label>
          <input type="checkbox"
                 name={{id}}
                 id={{id}}
                 {{ "checked" if active }}
          >
-         Active
+         <div class="checkbox--styled"></div>
+         Exclude gridded services
```

Comment:
Use `{{ label }}` here instead of inline text:

![image](https://user-images.githubusercontent.com/1430060/31082876-0d32941a-a75e-11e7-8c2b-e7129a989f7c.png)